R/filterReadsBam.R

```diff
@@ -0,0 +1,174 @@
+#' Write records from \code{infile} to \code{outfile} if they pass filter criteria.
```

Comment:
Maybe explicitly mention 'bam records' (and on line 3, 'For each bam file') for clarity?

Reply:
done in ab61c2fe